guides/rails_guides/markdown/renderer.rb

```diff
@@ -90,6 +94,19 @@ def clipboard_content(code, language)
             code = code.lines.grep(prompt_regexp).join.gsub(prompt_regexp, "")
           end
 
+          # Remove comments that reference an application file.
+          filepath_regexp =
+            case language
+            when "erb", "html+erb"
+              /^<%# #{APPLICATION_FILEPATH_REGEXP}.* %>/
+            when "ruby", "yaml", "yml"
+              /^# #{APPLICATION_FILEPATH_REGEXP}/
+            end
```

Comment:
Looks like Rubocop is not liking it 😅 

```
Offenses:

guides/rails_guides/markdown/renderer.rb:11:1: C: [Correctable] Layout/EmptyLinesAroundClassBody: Extra empty line detected at class body beginning.
guides/rails_guides/markdown/renderer.rb:101:15: C: [Correctable] Performance/ConstantRegexp: Extract this regexp into a constant, memoize it, or append an /o option to its options.
              /^<%# #{APPLICATION_FILEPATH_REGEXP}.* %>/
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
guides/rails_guides/markdown/renderer.rb:103:15: C: [Correctable] Performance/ConstantRegexp: Extract this regexp into a constant, memoize it, or append an /o option to its options.
              /^# #{APPLICATION_FILEPATH_REGEXP}/
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

3274 files inspected, 3 offenses detected, 3 offenses autocorrectable
```

Reply:
I've added the `/o` and extracted these regexps to constants.